### PR TITLE
[tesla] Removed dependency on org.apache.commons

### PR DESCRIPTION
**Pull request by cweitkamp**
- Removed dependency on org.apache.commons

Related to #7722

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaVehicleHandler.java [hunk 1]
```diff
@@ -991,7 +991,8 @@ protected boolean establishEventStream() {
                     eventClient = clientBuilder.build()
                             .register(new Authenticator((String) getConfig().get(CONFIG_USERNAME), vehicle.tokens[0]));
                     eventTarget = eventClient.target(URI_EVENT).path(vehicle.vehicle_id + "/").queryParam("values",
-                            StringUtils.join(EventKeys.values(), ',', 1, EventKeys.values().length));
+                            Arrays.asList(EventKeys.values()).stream().map(Enum::toString)
+                                    .collect(Collectors.joining(",")));
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaVehicleHandler.java:995]: Is this really the same? Or do we have to ignore the first element?

Comment (fwolter) [reply bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaVehicleHandler.java:995]: Seems like we need to `skip()` one.

Comment (cweitkamp) [reply bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaVehicleHandler.java]: Thanks. I added `skip(1)`.

**Review by J-N-K — APPROVED** [review]

**Review by kaikreuzer — APPROVED** [review]
Jenkins failure unrelated. Thanks!

**Pull request merged**